### ISSUE TITLE
Allow `Skipping Canary Suite` for `appveyor`

### ISSUE THREAD

**Issue by evilebottnawi** [webpack-contrib/webpack-defaults#124]
* Operating System: not require
* Node Version: not require
* NPM Version: not require
* webpack Version: not require
* webpack-defaults Version: latest

### Expected Behavior / Situation

Green CI on `appveyor.yml`.

### Actual Behavior / Situation

Failed CI on `appveyor.yml`.

### Modification Proposal

Same as here https://github.com/webpack-contrib/webpack-defaults/blob/master/templates/.circleci/config.yml#L25, we already support this for CircleCI

**Comment by michael-ciniawsky** [comment]
Agreed `appveyor` should run with `webpack@latest` only by default, the `canary` testing also never really took off and once `next` branch or  `webpack@next` gets merged into `master` or released as `@latest` we test 'old'  code as `canary` which is weird... So we should reconsider removing canary testing in general imho

**Comment by evilebottnawi** [comment]
@michael-ciniawsky canary is `next` tag, when `webpack@5` will be complied (unstable releases - all releases before `rc` are located in next tag), we can't remove this, we should skipping when next bracn is lower than stable as i written above.

**Comment by michael-ciniawsky** [comment]
kk open for suggestion how to implement this :)
